py/desiutil/test/test_install.py

```diff
@@ -624,26 +623,39 @@ def test_permissions(self, mock_chmod, mock_walk):
         self.desiInstall.install_dir = join(self.data_dir, 'desiutil')
         self.desiInstall.is_branch = False
         mock_walk.return_value = iter([(join(self.desiInstall.install_dir, 'bin'), [], ['executable', 'README.txt']),
-                                       (join(self.desiInstall.install_dir, 'lib', 'python3.10', 'site-packages', 'desiutil-1.2.3.dist-info'), [], ['METADATA', 'LICENSE.rst']),
-                                       (join(self.desiInstall.install_dir, 'lib', 'python3.10', 'site-packages', 'desiutil', '__pycache__'), [], ['__init__.cpython-3.10.pyc', 'module.cpython-3.10.pyc']),
-                                       (join(self.desiInstall.install_dir, 'lib', 'python3.10', 'site-packages', 'desiutil'), ['__pycache__'], ['__init__.py', 'module.py']),
-                                       (join(self.desiInstall.install_dir, 'lib', 'python3.10', 'site-packages'), ['desiutil-1.2.3.dist-info', 'desiutil'], []),
+                                       (join(self.desiInstall.install_dir, 'lib', 'python3.10', 'site-packages',
+                                             'desiutil-1.2.3.dist-info'), [], ['METADATA', 'LICENSE.rst']),
+                                       (join(self.desiInstall.install_dir, 'lib', 'python3.10', 'site-packages',
+                                             'desiutil', '__pycache__'), [], ['__init__.cpython-3.10.pyc', 'module.cpython-3.10.pyc']),
+                                       (join(self.desiInstall.install_dir, 'lib', 'python3.10', 'site-packages',
+                                             'desiutil'), ['__pycache__'], ['__init__.py', 'module.py']),
+                                       (join(self.desiInstall.install_dir, 'lib', 'python3.10', 'site-packages'),
+                                        ['desiutil-1.2.3.dist-info', 'desiutil'], []),
```

Comment:
Checking: will these tests work for python versions other than 3.10 (which is what we have at NERSC and appears to be the only thing in the test matrix) or is the test somehow mocking up 3.10 even if running under a different version?  If I'm parsing this correctly, the change here is only formatting so we don't necessarily need to fix it now even if this won't work for python >3.10, but I'll ask now while it's on radar.

Reply:
Ah, I see your point. Currently the test matrix includes 3.10 -- 3.13, and tests are passing, so I think the Python version is simulated in this case, but I'll take a closer look.

Reply:
The value `'python3.10'` is supplied as a return value to a mock object during testing, and then the value is confirmed in subsequent assert statements, so indeed the version is effectively simulated. However, to prevent confusion, I think I will allow that value to match the version of Python being used to prevent future confusion.